pkg/cfg/cfg.go

```diff
@@ -73,6 +73,7 @@ type oauthConfig struct {
 	ClientSecret    string   `mapstructure:"client_secret"`
 	AuthURL         string   `mapstructure:"auth_url"`
 	TokenURL        string   `mapstructure:"token_url"`
+	LogoutURL       string   `mapstructure:"logout_url"`
```

Comment:
I'm not sure but this might better be `end_session_endpoint`
https://developer.okta.com/docs/reference/api/oidc/#logout end_session_endpoint
https://openid.net/specs/openid-connect-session-1_0-17.html

WDYT?

Reply:
Based on the precedent from `configureOAuthClient`, he existing nomenclature is that individual API endpoints are identified by things named `*URL`, and that a collection of related API endpoints is called `Endpoint`. So I would keep `_url` here for consistency, but I think that `end_session_url` would be fine. WDYT?